js/copy-service/Substitutions/Substitutions.test.ts

```diff
@@ -80,7 +91,8 @@ describe('Substitutions', () => {
       expect(substitutions.get).toHaveBeenCalledWith(key);
     });
 
-    const testValue = (testMessage, value, expectedResult) => {
+    // eslint-disable-next-line @stylistic/max-len
+    const testValue = (testMessage: string, value: number | boolean | string | (() => number) | null, expectedResult: boolean) => {
```

Comment:
We generally do not want to disable linter rules - the rules are there for a reason. The `no-explicit-any` rule is a case-by-case basis right now, but other rules should be followed.
```suggestion
    const testValue = (
      testMessage: string, 
      value: number | boolean | string | (() => number) | null, 
      expectedResult: boolean
    ) => {
```